padl/transforms.py

```diff
@@ -565,14 +606,24 @@ def _pd_itercall(self, args, mode: Mode, loader_kwargs: Optional[dict] = None,
             else:
                 output = batch
 
-            if use_post:
-                output = post.pd_call_transform(output, mode)
+            # if use_post:
+                # output = post.pd_call_transform(output, mode)
 
             if flatten:
+
                 if verbose:
                     pbar.update()
-                if not use_post:
-                    output = Unbatchify(cpu=False)(batch)
+
+                # TODO this was edited
+                if use_post:
```

Comment:
`post.pd_call_transform` is now hidden under the `flatten` flag. Was this intended?

Reply:
This means the entire post step could be skipped if the user doesn't specify `flatten=True`

Reply:
Sorry, I didn't see the `assert` above.

Reply:
Yes all intended